api/polaris-catalog-service/build.gradle.kts

```diff
@@ -81,10 +81,12 @@ openApiGenerate {
   ignoreFileOverride = "$rootDir/.openapi-generator-ignore"
   removeOperationIdPrefix = true
   templateDir = "$rootDir/server-templates"
-  globalProperties.put("apis", "GenericTableApi,PolicyApi")
-  globalProperties.put("models", models)
-  globalProperties.put("apiDocs", "false")
-  globalProperties.put("modelTests", "false")
+  // TODO -- revert the changes to these values after the Iceberg 1.8 change has merged
```

Comment:
I'm able to run `openApiGenerate` to generate everything, but I think that these gradle changes are needed in order for CI to generate the new types?

See #1037, where [an attempt to partially pull in rest spec changes](https://github.com/apache/polaris/pull/1037/files#diff-a3256d309d2321d357af75f2dcf679e9d1ddfba982a3b4e6664702b4b3f44eeaR937) resulted in CI failures as generated classes weren't remade.